tests/vmi_lifecycle_test.go

```diff
@@ -1692,24 +1708,25 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 		})
 		Context("with ACPI and some grace period seconds", func() {
 			DescribeTable("[rfe_id:273][crit:medium][vendor:cnv-qe@redhat.com][level:component]should result in vmi status succeeded", func(gracePeriod int64) {
-				vmi = newCirrosVMI()
+				var vmi *v1.VirtualMachineInstance
 
 				if gracePeriod >= 0 {
-					vmi.Spec.TerminationGracePeriodSeconds = &gracePeriod
+					vmi = libvmi.NewCirros(libvmi.WithTerminationGracePeriod(gracePeriod))
 				} else {
 					gracePeriod = v1.DefaultGracePeriodSeconds
+					vmi = libvmi.NewCirros()
 					vmi.Spec.TerminationGracePeriodSeconds = nil
 				}
 
 				By("Creating the VirtualMachineInstance")
-				obj, err := virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(vmi)
+				vmi, err := virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(vmi)
```

Comment:
It does not seem related to this commit.

Reply:
It's a small fixup that for simplicity I've included in the commit.

Reply:
I think that it will help borth the review and the cleanness of the history to place this in a separate commit.